meta_review/models.py

```diff
@@ -0,0 +1,77 @@
+from django.db import models
+
+
+class Participant(models.Model):
+    login = models.TextField(default=None, primary_key=True)
+    name = models.TextField(default=None, null=True)
+    score = models.FloatField(default=0, null=True)
+    rank = models.IntegerField(default=None, null=True)
+
+    # ranking trend compared to last iteration
+    trend = models.IntegerField(default=None, null=True)
+
+    # time of latest action
+    last_active_at = models.DateTimeField(default=None, null=True)
+
+    # number of positive reactions received
+    pos_in = models.IntegerField(default=0, null=True)
+
+    # weighted positive reactions received
+    weighted_pos_in = models.FloatField(default=0, null=True)
+
+    # number of positive reactions give away
+    pos_out = models.IntegerField(default=0, null=True)
+
+    # number of negative reactions received
+    neg_in = models.IntegerField(default=0, null=True)
+
+    # weighted negative reactions received
+    weighted_neg_in = models.FloatField(default=0, null=True)
+
+    # number of negative reactions give away
+    neg_out = models.IntegerField(default=0, null=True)
+
+    # punishment points
+    punishment = models.FloatField(default=0, null=True)
+
+    # weight factor
+    weight_factor = models.FloatField(default=0.1, null=True)
+
+    def __str__(self):
+        return 'Meta-reviewer: ' + self.login
+
+    class Meta:
+        ordering = ['rank']
+
+
+class Comment(models.Model):
+    id = models.TextField(default=None, primary_key=True)
```

Comment:
Could you consider renaming `id` to `identifier` ? `id` would interfere with python's built-in function.

Reply:
I would like to hear more voice.

I have thought it for a while and tried finding some evidence to support my original idea. I did found some, very weak though.

As https://docs.djangoproject.com/en/2.0/topics/db/models/#automatic-primary-key-fields points out,

By default, Django gives each model the following field:

> id = models.AutoField(primary_key=True)

Also, from https://github.com/django/django/blob/f1f4aeb22e7bc9b504f69f7cb111ac9bdedb5f1e/django/db/models/base.py#L1298-L1312 we can see that they allow the usage of `id` field as long as it has `primary_key=True`

Reply:
@hemangsk could you step in ;)

I'm pretty sure `id` is ok, but I'm not a django master.

Reply:
It seems alright to me as well.